conda/faiss/build-lib.sh

```diff
@@ -7,18 +7,18 @@
 set -e
 
 
-# Build libfaiss.so/libfaiss_avx2.so/libfaiss_avx512.so
+# Build libfaiss.so/libfaiss_avx2.so/libfaiss_avx512.so/libfaiss_avx512_sr.so
 cmake -B _build \
       -DBUILD_SHARED_LIBS=ON \
       -DBUILD_TESTING=OFF \
-      -DFAISS_OPT_LEVEL=avx512 \
+      -DFAISS_OPT_LEVEL=avx512-sr \
       -DFAISS_ENABLE_GPU=OFF \
       -DFAISS_ENABLE_PYTHON=OFF \
       -DBLA_VENDOR=Intel10_64lp \
       -DCMAKE_INSTALL_LIBDIR=lib \
       -DCMAKE_BUILD_TYPE=Release .
 
-make -C _build -j$(nproc) faiss faiss_avx2 faiss_avx512
+make -C _build -j$(nproc) faiss faiss_avx2 faiss_avx512 faiss_avx512_sr
```

Comment:
This is used for faiss's conda packaging upload. I don't think we want to expose this build mode yet in conda officially. Can you omit this for now?

Reply:
Removed.